11_FFTBloom/app_resources/fft_convolve_ifft.hlsl

```diff
@@ -0,0 +1,205 @@
+#include "common.hlsl"
+
+/*
+ * Remember we have these defines:
+ * _NBL_HLSL_WORKGROUP_SIZE_
+ * ELEMENTS_PER_THREAD
+ * KERNEL_SCALE
+ * scalar_t
+ * FORMAT
+*/
+
+// Can't specify format
+/* [[vk::combinedImageSampler]]*/ [[vk::binding(0, 0)]] /* [[vk::image_format(FORMAT)]]*/ Texture2D<float32_t2> kernelChannels[CHANNELS] : register(t0);
+/* [[vk::combinedImageSampler]]*/ [[vk::binding(0, 0)]] SamplerState samplerState[CHANNELS] : register(s0);
```

Comment:
remove the `: register()` bullcrap, its redundant `vk::binding(0,0)` does all the work

P.S. also what's with the `combinedImageSampler` and the format stuff?